docs/compiler/Introduction.md

```diff
@@ -0,0 +1,57 @@
+<!-- README Document on how to build and use this Neural Network Compiler -->
+
+# Introduction
+
+This document discusses the ONE compiler. ONE compiler analyses, quantizes and optimizes the input model. It takes many model formats and converts to a common [Circle](#circle-format)  format which is then deployed.
+
+<!-- Contents -->
+# Contents 
+* [Build Setup](#Build-Setup)
+* [Supported Formats](#Supported-Formats)
+* [Circle Format](#Circle-Format)
+
+<!-- Build Setup -->
+## Build Setup
+<!-- System Requirments -->
+* System Setup   
+This setup is to build the ONE compiler on Ubuntu. Its tested on Ubuntu 16.04 as well as Ubuntu 18.04.  
+Get the code [Samsung ONE](https://github.com/Samsung/ONE) from Github. 
+
+<!-- Software/Package Requirements -->
+* Dependencies
+    * Cmake (Greater than or equal to 3.1)
+    * g++ toolchain (Greater than or equal to 6)
+    * python virtual environment (python3-venv for python3)
+    * HD5 library (libhdf5-dev)
```

Comment:
You mean `HDF5` ?

Reply:
@llFreetimell , is `boost` needed? or will it be replaced?

Reply:
hmm...? According to my brief search, we may need `boost` but I am not sure :(

Reply:
Sorry :) @mhs4670go ,  is boost needed? or will it be replaced?

Reply:
@seanshpark nnc is using `boost`.

Reply:
@mhs4670go and @llFreetimell would n't ./nncc configure take care of rest of the packages including boost?